File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) 
    {
        set<int> mp;
        for(int i=0;i<nums.size();i++)
        {
            if(mp.find(nums[i])!=mp.end())
                return true;
            else
                mp.insert(nums[i]);
        }
        return false;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        n = len(nums)
        cov_range = 0
        for i in range(n):
            if i<=cov_range:
                if i == n-1:
                    return True
                cov_range = max(cov_range, nums[i]+i)
            else:
                return False